string name
string graspit_file
string moveit_file
geometry_msgs/PoseStamped pose